app/models/vmdb_table.rb

```diff
@@ -6,6 +6,35 @@ class VmdbTable < ApplicationRecord
 
   has_one  :latest_hourly_metric,  -> { VmdbMetric.where(:capture_interval_name => 'hourly', :resource_type => 'VmdbTable', :timestamp => VmdbMetric.maximum(:timestamp)) }, :as => :resource, :class_name => 'VmdbMetric'
 
+  # index_name1,unique1,col11,col12|index_name2,unique2,col21,col22
+  virtual_attribute :all_indexes, :string, :arel => (lambda do |t|
+    t.grouping(
+      Arel::Nodes::SqlLiteral.new(
+        "SELECT
+           string_agg(
+           distinct ix.relname || ',' || indisunique || ',' ||
+          regexp_replace(pg_get_indexdef(indexrelid), '^[^\\)]*\\(([^\\)]*)\\).*$', '\\1'), '|')
```

Comment:
For the record, I ended up testing this in ruby, and was a bit surprised by the results:

```ruby
irb(main):001:0> "foo bar (match)".match /^[^\)]*\(([^\)]*)\).*$/
=> #<MatchData "foo bar (match)" 1:"match">
irb(main):002:0> "foo bar (match)".match /^[^\(]*\(([^\)]*)\).*$/
=> #<MatchData "foo bar (match)" 1:"match">
irb(main):003:0> "(foo bar)(match)".match /^[^\(]*\(([^\)]*)\).*$/
=> #<MatchData "(foo bar)(match)" 1:"foo bar">
irb(main):004:0> "(foo bar)(match)".match /^[^\)]*\(([^\)]*)\).*$/
=> #<MatchData "(foo bar)(match)" 1:"foo bar">
```

Didn't seem to matter what the first parens was, it just matched it just fine... so yeah... maybe just add some more specs around it instead of changing? (in case it changes with psql versions)